src/commands/Tools/Dictionary/define.ts

```diff
@@ -0,0 +1,82 @@
+import { KlasaMessage } from 'klasa';
+import { TOKENS } from '@root/config';
+import { DbSet } from '@lib/structures/DbSet';
+import { RichDisplayCommand, RichDisplayCommandOptions } from '@lib/structures/RichDisplayCommand';
+import { UserRichDisplay } from '@lib/structures/UserRichDisplay';
+import { cutText, fetch, FetchResultTypes } from '@utils/util';
+import { toTitleCase } from '@klasa/utils';
+import { MessageEmbed } from 'discord.js';
+import { ApplyOptions } from '@skyra/decorators';
+import { BrandingColors } from '@utils/constants';
+
+const options = { headers: { Accept: 'application/json', Authorization: `Token ${TOKENS.OWLBOT}` } };
```

Comment:
Just inline the options below, We don't really set it outside in other commands either. Also as Kyra said in Discord, use the Mime.Types enum for the accept.